content/en/blog/_posts/2021-00-00-kpng-specialized-proxiers.md

```diff
@@ -0,0 +1,237 @@
+---
+layout: blog
+title: "Use KPNG to write specialized kube-proxiers"
+date: 2021-09-00
+slug: use-kpng-to-write-specialized-kube-proxiers
+---
+
+Authors: Lars Ekman, Ericsson
+
+The post describes how you can create a specialized proxier using
+Kubernetes Proxy NG (kpng) without interfering with the existing
```

Comment:
link here https://github.com/kubernetes-sigs/kpng???

Reply:
I am a bit unsure if links are allowed in the initial paragraph. No other blogs seem to have them.
@sftim Is it allowed?